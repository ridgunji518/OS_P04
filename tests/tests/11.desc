Error Test 1: Bad vmalloc flag 
