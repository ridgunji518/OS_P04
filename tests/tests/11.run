cd ../solution; ../tests/run-xv6-command.exp CPUS=4 Makefile.test test_11 | grep -E 'XV6_TEST_OUTPUT|XV6_TEST_ERROR'; rm *test_11* ;cd ../tests
