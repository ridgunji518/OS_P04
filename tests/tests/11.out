XV6_TEST_OUTPUT Please pass VMALLOC_SIZE_BASE or VMALLOC_SIZE_HUGE as flag.
